client/my-sites/site-settings/seo-settings/test/help.jsx

```diff
@@ -1,23 +1,38 @@
 /**
  * @jest-environment jsdom
  */
-import { shallow } from 'enzyme';
-import { SeoSettingsHelpCard } from '../help';
+import { FEATURE_ADVANCED_SEO } from '@automattic/calypso-products';
+import { screen } from '@testing-library/react';
+import { reducer as ui } from 'calypso/state/ui/reducer';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
+import SeoSettingsHelpCard from '../help';
 
-const props = {
-	disabled: false,
-	siteId: 1,
-	translate: ( x ) => x,
+const siteId = 1;
```

Comment:
Seeing `siteId: 1` is scary even in tests. Can we at least use some arbitrary random number here?

Reply:
Sure thing!